pwnlib/constants/darwin/aarch64.py

```diff
@@ -0,0 +1,1070 @@
+from pwnlib.constants.constant import Constant
+STDIN_FILENO = Constant('STDIN_FILENO',0)
```

Comment:
file was copied from freebsd.py, I need write this file manualy? 
Most of syscalls in darwinbsd and freebsd looks same

Reply:
https://opensource.apple.com/source/xnu/xnu-1228/bsd/sys/syscall.h.auto.html
https://github.com/apple/darwin-xnu/blob/main/bsd/kern/makesyscalls.sh

Reply:
Ok, we should download latest macosSdk ( from https://github.com/nullgemm/instant_macos_sdk ).
Then generate file using: `./Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/sys/syscall.h`